app/brave_main_delegate.cc

```diff
@@ -225,6 +228,10 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
     enabled_features.insert(features::kDnsOverHttps.name);
   }
 
+  if (chrome::GetChannel() == version_info::Channel::CANARY) {
```

Comment:
not sure if our nightly maps to `CANARY`, does it? @bridiver

Reply:
hopefully soon we will be able to do such things via variations service

Reply:
I found that mapping to be consistent in a few places in the codebase, e.g.
- https://github.com/brave/brave-core/blob/f9e6cdd714b9e71ee53d132875efb73042d94f87/chromium_src/chrome/install_static/brave_install_util_unittest.cc#L594
- https://github.com/brave/brave-core/blob/f9e6cdd714b9e71ee53d132875efb73042d94f87/common/brave_channel_info_posix.cc#L34-L35
- https://github.com/brave/brave-core/blob/f9e6cdd714b9e71ee53d132875efb73042d94f87/common/brave_channel_info_unittest.cc#L74-L75